static/js/leaflet-heat.js

```diff
@@ -0,0 +1,112 @@
+/*
+ (c) 2014, Vladimir Agafonkin
+ simpleheat, a tiny JavaScript library for drawing heatmaps with Canvas
+ https://github.com/mourner/simpleheat
+*/
+!(function() {
```

Comment:
This file (and all the html/css/js files) will be served up through flask. So, it should probably live inside the python flask app (the application folder inside this repo).

Reply:
Let's take some time to walk through this tonight—I want to be sure I understand all the dependencies at play.